src/helpers/mergeArrays.js

```diff
@@ -19,7 +19,10 @@ const createMergeArrays = mergeItemFn => (prev, next) => {
 
 const mergeArrays = createMergeArrays((prev, next) => next)
```

Comment:
maybe move that at the bottom of the file?

Reply:
not sure why, but ok :)